src/m365/pp/commands/copilot/copilot-get.ts

```diff
@@ -88,22 +92,23 @@ class PpChatbotGetCommand extends PowerPlatformCommand {
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    await this.showDeprecationWarning(logger, "pp chatbot get", "pp copilot get");
```

Comment:
Let's use the command names from the command file `commands.CHATBOT_GET` and `commands.COPILOT_GET`.